docs/release-payload.md

```diff
@@ -0,0 +1,112 @@
+# Publishing a KNI Release Payload
+
+OpenShift publishes a release payload image which includes information
+about cluster operator images and their resource manifests, along with
+references to installer and CLI images. The recommended method for
+obtaining an installer binary is to first choose a release version and
+then use the `oc adm release extract --tools` command to extract the
+installer binary from the release payload.
+
+Since KNI has temporarily forked the installer, we build custom and
+publish custom release payloads that include a reference to the forked
+installer.
+
+## Preparation and Configuration
+
+We build and publish within a namespace on an OpenShift
+cluster. First, prepare a `kubeconfig` with credentials to this
+cluster, and with the desired namespace set as the default:
+
+```
+$ oc --config=release-kubeconfig login https://api.ci.openshift.org --token=...
+$ oc --config=release-kubeconfig new-project kni
+$ oc --config=release-kubeconfig project kni
+$ oc --config=release-kubeconfig adm policy add-role-to-user admin <other admin>
```

Comment:
Who is `<other admin>`? Is this to give someone else access?

Reply:
Yeah - e.g. I gave @russellb admin access (or he gave me, I forget)